### PR TITLE
add take-[] and drop-[] to stdl Data/List/Properties.agda

### PR DESCRIPTION
**Pull request by Sofia-Insa**
Add lemmas `take-[]` and `drop-[]`  to std library. In Data.List.Properties.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2784,6 +2784,12 @@ Other minor changes
   foldr-map : foldr f x (map g xs) ≡ foldr (g -⟨ f ∣) x xs
   foldl-map : foldl f x (map g xs) ≡ foldl (∣ f ⟩- g) x xs
   ```
+  
+ * Added new lemmas in `Data.List.Proppreties`:
+  ```
+  take-[] : ∀ m → take {A = A} m [] ≡ []
```
Comment (JacquesCarette) [inline CHANGELOG.md:2790]: Don't need the `{A = A}` in the CHANGELOG.

**Review by JacquesCarette — CHANGES_REQUESTED** [review]

**Review by JacquesCarette — APPROVED** [review]

File: src/Data/List/Properties.agda [hunk 2]
```diff
@@ -758,6 +758,11 @@ length-take zero    xs       = refl
 length-take (suc n) []       = refl
 length-take (suc n) (x ∷ xs) = cong suc (length-take n xs)
 
+-- Taking and dropping from an empty list do nothing.
```
Comment (MatthewDaggitt) [inline src/Data/List/Properties.agda:761]: This comment should only reference `taking`, likewise the one below only `dropping`.

File: src/Data/List/Properties.agda [hunk 3]
```diff
@@ -758,6 +758,11 @@ length-take zero    xs       = refl
...
+take-[] : ∀ m → take {A = A} m [] ≡ []
+take-[] ℕ.zero = refl
```
Comment (MatthewDaggitt) [inline src/Data/List/Properties.agda:763]: Can get rid of the `ℕ.` here and below as well.

**Review by MatthewDaggitt — COMMENTED** [review]
Otherwise looks great! Thanks for the first PR.

File: agda-stdlib
Comment (gallais) [inline agda-stdlib:1]: ugh?

Comment (Sofia-Insa) [reply agda-stdlib:1]: Sorry, I got mixed up when I tried to do a new PR changes end up here. Didn't manage to delete a commit so I commit once more to delete the changes. 
There is only the add of take-[] and drop-[] in Data.List.Propreties and change in CHANGELOG.md that should appear now

**Comment by MatthewDaggitt** [comment]
Err so it looks like now you've added the library as a git submodule of itself and then made the changes  in the submodule rather than the main library?!

**Pull request merged**